src/itinerary/index.tsx

```diff
@@ -27,49 +26,42 @@ const Itinerary = ({
     {
       showFromDistance && (
         <li className="kirk-itinerary-fromDeparture">
-          <span>{departure.distanceFromPoint}</span>
+          <span>{places[0].distanceFromPoint}</span>
         </li>
       )
     }
-    <li className="kirk-itinerary-location kirk-itinerary--departure">
-      {
-        !small && (
-          <time dateTime={departure.isoDate}>{departure.time}</time>
-        )
-      }
-      <div>
-        <span>{departure.mainLabel}</span>
-        {
-          (!small && departure.subLabel) && (
-            <span className="kirk-itinerary-subtext">{departure.subLabel}</span>
-          )
-        }
-      </div>
-    </li>
-    <li
-      className={cc([
-        'kirk-itinerary-location kirk-itinerary--arrival',
-        { 'kirk-itinerary-location--fromArrival': showToDistance },
-      ])}
-    >
-      {
-        !small && (
-          <time dateTime={arrival.isoDate}>{arrival.time}</time>
-        )
-      }
-      <div>
-        <span>{arrival.mainLabel}</span>
-        {
-          (!small && arrival.subLabel) && (
-            <span className="kirk-itinerary-subtext">{arrival.subLabel}</span>
-          )
-        }
-      </div>
-    </li>
+    {
+      places.map((place, index) => (
```

Comment:
So you display the first and the last twice ?

Why not:

const [first, rest, last] = places

Reply:
I don't see how it could be displayed twice?

Reply:
@pith This wouldn't work :)
`const [first, rest, last] = places`